site/content/help/feature/multiplayer/index.md

```diff
@@ -55,12 +56,14 @@ You can also save this command (with arguments filled in) as a script for easier
 If you are on Windows, it is also possible to host rooms using a standalone executable. As you may already know, every *Citra* release comes with three executables - `citra` (The command line emulator frontend), `citra-qt` (The GUI emulator frontend) and `citra-room`, the dedicated room server which we'll use.
 
 Steps:
+
 1. If you do not have a *Citra* installation yet, install it with the installer from our [official website](https://citra-emu.org).
```

Comment:
I don't know how this went unnoticed all this while, but can you fix the numbering issue that originates here?

Reply:
Markdown automatically numbers numbered lists. Therefore, by always writing 1. you won't need to change anything if you ever want to add an item in the middle.